pkg/controller/perconaservermongodb/backup.go

```diff
@@ -186,3 +185,29 @@ func (r *ReconcilePerconaServerMongoDB) isBackupRunning(cr *api.PerconaServerMon
 
 	return false, nil
 }
+
+func (r *ReconcilePerconaServerMongoDB) updatePBMConfig(cr *api.PerconaServerMongoDB) error {
+	isRestoring, err := r.isRestoreRunning(cr)
+	if err != nil {
+		return fmt.Errorf("checking if restore running on pbm update: %w", err)
```

Comment:
Please use `errors.Wrap()` since we are using it everywhere in new code parts and old `fmt` style errors will be refactored

Reply:
Ok, I'll change it